.gitignore

```diff
@@ -3,6 +3,9 @@
 # active project is stored in this file
 active-project
 
+*.sublime-workspace
```

Comment:
```
Projects in Sublime Text are made up of two files: the .sublime-project file, which contains the project definition, and the .sublime-workspace file, which contains user specific data, such as the open files and the modifications to each. 
```

https://www.sublimetext.com/docs/3/projects.html

Reply:
goto workspace?

Reply:
yeah, I added a sublime project for goto, in order to make the sublime Goto (cmd + P) not list entries in the .tox folder.  It annoyed me that jumping to different files involved scrolling down on the list to select the right source file. 

Ofcourse sublime then adds a workspace file which is highly personal and only relevant to the current sublime session. It is recommended to not check in to source control.